ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -433,23 +462,34 @@ Query params for the authorisation request are given below:
   <tr>
    <td><code>authorization_details</code>
    </td>
-   <td>As specified in OAuth 2.0 Rich Authorization Requests specification to specify fine-grained access [4]. An example is as given below:
+   <td>As specified in OAuth 2.0 Rich Authorization Requests specification to specify fine-grained access [4]. 
+   
+   An example of W3C VC credential format is as given below:
 
    ```json
-   {
-      "type": "openid_credential",
-      "locations": [
-         "https://credential-issuer.example.com"
-      ],
-      "format": "jwt_vc_json",
-      "credential_definition": {
-         "type": [
-            "VerifiableCredential",
-            "UniversityDegreeCredential"
-         ]
-      }
-   }
+    [
+        {
+            "type": "openid_credential",
+            "credential_configuration_id": "VerifiablePortableDocumentA1",
+            "credential_definition": {
+                "credentialSubject": {}
+            }
+        }
+    ]
    ```
+
+  The IETF SD-JWT VC is as given:
+
+   ```json
+  [
+      {
+          "type": "openid_credential",
+          "format": "vc+sd-jwt",
+          "vct": "VerifiablePortableDocumentA1"
+      }
+  ]
+  ```
+
    > [!NOTE]
    > You may also use the earlier version as supported by EBSI.
```

Comment:
If I understood correctly, this RFC provides three different ways how the `authorization_details` can be presented. Could that create problems in the implementation phase?